receiver/kafkametricsreceiver/metadata.yaml

```diff
@@ -30,6 +39,13 @@ metrics:
       monotonic: false
       value_type: int
       aggregation_temporality: cumulative
+  kafka.broker.log_retention_hours:
+    enabled: true
```

Comment:
We generally prefer to not enable metrics by default. I see all of the added metrics are enabled by default, can you share justification as to why they need to be enabled by default? (For more context, it's considered a breaking change to add any new metrics that will be enabled by default).

Reply:
Thanks for the context, It's fine if we keep these new metrics disabled by default. I have made necessary changes.